trade-finance-manager-ui/templates/utilisation-reports/find-utilisation-reports-by-year.njk

```diff
@@ -2,31 +2,27 @@
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/input/macro.njk" import govukInput %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
-
 {% extends "index.njk" %}
-
 {% set backLink = { href: "/utilisation-reports" } %}
-
 {% block pageTitle %}
   Find reports by year
 {% endblock %}
 {% block content %}
   <div class="govuk-grid-row govuk-!-padding-top-7">
-    <div
-      class="govuk-grid-column-full">
+    <div class="govuk-grid-column-full">
       {% if errorSummary | length %}
         {{ govukErrorSummary({
           titleText: "There is a problem",
           errorList: errorSummary,
           attributes: {
-            'data-cy': 'error-summary'
+            "data-cy": "error-summary"
           }
         }) }}
       {% endif %}
       <h1 class="govuk-heading-xl govuk-!-margin-bottom-6" data-cy="find-reports-by-year-heading">
         Find reports by year
       </h1>
-      <form method="GET" data-cy="form">
+      <form method="GET" data-cy="form" id="find-utilisation-reports-by-year--form">
```

Comment:
We haven't used `id` attribute in form tag, can `data-cy` attribute rather please be used?

Reply:
The `id` attribute is used in the public script to select this specific form, and using `data-cy` as a selector for your public facing code is not best practice

Reply:
Understood thanks for explaning.